oasis/access-control.md

```diff
@@ -1,39 +1,48 @@
 ---
 layout: default
 description: This guide explains which access control concepts are available in Oasis and how to use them.
-title: Access control in ArangoDB Oasis
+title: Security and access control in ArangoDB Oasis
 ---
-# Access control in ArangoDB Oasis
+# Security and access control in ArangoDB Oasis
 
-Oasis has a structured set of resources that are subject to
+Oasis has a structured set of resources that are subject to security and
 access control:
 
 - Organizations
 - Projects
 - Deployments
 
 For each of these resources you can perform various operations.
-For example, you can *create* a project in an organization. This requires the
-respective **permission**, which can be granted to organization members via a
-**role** such as the pre-defined *Project Administrator*.
+For example, you can create a project in an organization and create a deployment inside a project.
 
-The association of a member with a role is called a **role binding**.
-All role bindings of a resource comprise a **policy**.
+## Locked resources
+
+In ArangoDB Oasis, you can lock the resources and some of their properties to prevent accidental deletion. When an object is locked,
```

Comment:
Please remove "and some of their properties"

"When an object is locked" -> "When a resource is locked"